twill-yarn/src/main/hadoop20/org/apache/twill/internal/yarn/Hadoop20YarnAppClient.java

```diff
@@ -211,7 +218,6 @@ public YarnApplicationReport getReport() {
       try {
         return new Hadoop20YarnApplicationReport(yarnClient.getApplicationReport(appId));
       } catch (YarnRemoteException e) {
-        LOG.error("Failed to get application report {}", appId, e);
```

Comment:
removing the logging here is good. However, the messages could add useful context (here: the appId). If you wrap it into a RuntimeException, why not add the message?